packages/react-native-jigsaw/scripts/parser.js

```diff
@@ -1,10 +1,8 @@
-const fs = require("fs");
+const { promises: fs } = require("fs");
```

Comment:
seems like a node v15 & above thing? is that true?

if so then we def need to bump our nodejs version in the CI

didn't know about this, is awesome!

Reply:
`require("fs/promises")` is from Node v14, but the namespace promises have been since 12 I believe.